all_test.go

```diff
@@ -24,8 +25,11 @@ Suggested test runs that you should do:
 
 	go test github.com/IMQS/authaus -test.cpu 2
 	go test github.com/IMQS/authaus -test.cpu 2 -backend_postgres
+	go test github.com/IMQS/authaus -test.cpu 2 -backend_ldap
+	go test github.com/IMQS/authaus -test.cpu 2 -backend_ldap -backend_postgres
 	go test -race github.com/IMQS/authaus -test.cpu 2
 	go test -race github.com/IMQS/authaus -test.cpu 2 -backend_postgres
+	go test -race github.com/IMQS/authaus -test.cpu 2 -backend_ldap
```

Comment:
I think there should be some comments describing what happens under the various combinations of backend_postgres, backend_ldap. ie a matrix of what the test setup looks like with each of those flags on and off.